app/scripts/TrackRenderer.jsx

```diff
@@ -365,7 +663,8 @@ class TrackRenderer extends React.Component {
       const trackObject = this.trackDefObjects[track.track.uid].trackObject;
       trackObject.rerender(options);
 
-      if (track.track.hasOwnProperty('contents')) {
+      if (isCombinedTrackConfig(track.track)) {
```

Comment:
This special handling of combined tracks is all over. I tried to make a type utility that we can use consistently, and will effectively narrow the type for typescript.

Reply:
`CombinedTrack` is a strange track. It is the only Track doesn't extend `Track` that I can tell.

Reply:
yes, my understanding is that it’s a container for multiple inner tracks.